custom_components/tuya_local/devices/ferroli_titano_twin.yaml

```diff
@@ -0,0 +1,64 @@
+name: Ferroli Titano Twin
+products:
+  - id: lci0ebsz7ftaruaf
+    name: DT EWH
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: preset_mode
+          conditions:
+            - dps_val: manual
+              value: heat_cool
+            - dps_val: eco
+              value: heat
+    - id: 2
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: eco
```

Comment:
There is no point to have a preset mode with only one option.  It is essentially hardcoded.

Reply:
Water heater does not have a preset_mode, so the mapping can be removed and `hidden: true` added in its place, as all control will be through operation_mode.